src/main/resources/db/migration/V1_27__RECOMMENDATION_DOCUMENT_TABLE.sql

```diff
@@ -0,0 +1,4 @@
+alter table recommendation_document
```

Comment:
Don't we need to drop the current binary column from this table?

Reply:
This story is just about upload, not fetching/deleting. If I remove the binary column it will break the current implementation.